examples/full/src/components/header/index.js

```diff
@@ -2,6 +2,7 @@ import { h, Component } from 'preact';
 import { Link } from 'preact-router/match';
 import style from './style';
 
+// eslint-disable-next-line
```

Comment:
I was talking to @developit regarding this, he said he will soon add one. But I took care of the comment in another PR (up-coming). Added an extra set of rules to the `eslintConfig`